From e899beddf83faf0b80e98fb76274ad5cd6e6c651 Mon Sep 17 00:00:00 2001
From: Bin Yang <yangbin@rock-chips.com>
Date: Wed, 15 Feb 2017 10:45:08 +0800
Subject: [PATCH] arm64: dts: rockchip: fix dp register fail for rk3399-mid

1. MID board is only one dp port, we need to assign a phy.
2. MID board not use HDMI, we need to disabled it.

Change-Id: I589373fa04b940681910b3c0ee58ee9f9f464916
Signed-off-by: Bin Yang <yangbin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-next.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-next.dts b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-next.dts
index 533caa4c03ec..30cd20d7b381 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-next.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-next.dts
@@ -1008,9 +1008,14 @@
 	status = "disabled";
 };
 
+&hdmi {
+	status = "disabled";
+};
+
 &cdn_dp {
 	status = "okay";
 	extcon = <&fusb0>;
+	phys = <&tcphy0_dp>;
 };
 
 &dp_in_vopb {
-- 
2.35.3

